src/utils/mapUtils.ts

```
@@ -1,6 +1,6 @@
 import { RegionOption } from '../types/RegionDataTypes'
 import { regionOptions } from '../data/regionData'
-import { Map, MapGeoJSONFeature, MapLayerMouseEvent, LngLatBounds } from 'maplibre-gl'
+import { LngLatBounds, Map, MapGeoJSONFeature, MapLayerMouseEvent } from 'maplibre-gl'
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)